### PR TITLE
Code reviews

### PR DESCRIPTION
**Pull request by mikeclimbrock**

### CODE REVIEW

File: CodeReviews.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+[Shared Repositories](#shared-repositories)  
+[Individual Repositories](#individual-repositories)  
+[Document Changes](#document-changes)  
+
+## Shared Repositories
+*This includes all shared libraries (i.e. lib-play, lib-postgresql)*
+- A Pull Request is required
+- At least one (1) "Looks Good To Me (LGTM)" comment is required before merging to master branch
+
```
Comment (mbryzek) [inline CodeReviews.md]: Should we defined "shared" as a project that is used by two or more other projects at Flow?

File: CodeReviews.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## Individual Repositories
+- A Pull Request is requred so that a notification is sent to the Slack #pullrequests channel for awareness
```
Comment (mbryzek) [inline CodeReviews.md]: requred => required

File: CodeReviews.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- Pull Request may be merged without additional review and at authors convenience
```
Comment (mbryzek) [inline CodeReviews.md]: authors => author's

**Comment by mbryzek** [comment]
LGTM

**Comment by paololim** [comment]
Should we put this into Contribution.md? https://github.com/flowcommerce/standards/blob/master/Contributing.md

**Pull request merged**